SalesforceFoundation/EDA
# W-035944 - Add new custom flow and custom tasks to enable Shield Platform Encryption

A new custom flow and three new custom tasks were developed for EDA in order to setup Shield Platform Encryption more easily in a developer scratch org.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
1. Create a new developer scratch org.
2. After deploying EDA to the scratch org (using the `dev_org` or `qa_org` flow), run the `setup_encryption` flow.
3. Open the org and navigate to *Setup | Platform Encryption | Encryption Policy | Encrypt Fields*
4. Enable all standard fields for encryption.
    a. Optionally, enable EDA custom fields for encryption.
5. Run the EDA unit tests. Confirm they all pass.

(See Quip doc in WI for more details)
